tests/cypress/e2e/archiving-assets.cy.ts

```diff
@@ -0,0 +1,46 @@
+it('Assets / query params determine which asset is served', () => {
+  cy.visit('/asset-paths/query-params');
+});
+
+// TODO: Uncomment when Cypress support achieves parity with Playwright
```

Comment:
Maybe it would be better to leave this test running so we see it go from broken to working when we fix it.

Reply:
I guess the issue is it's not just a UI failure (showing broken image), it's a test run failure (the `test:cypress` command fails). Another option is I could skip the tests, so they are commented in and presumably show during each run (sort of a mild prompt to not skip them forever).

Reply:
Ah I didn't realize the cypress command would fail. Then yeah, skipping them might be slightly better but either way!